src/ripple/app/misc/impl/ValidatorSite.cpp

```diff
@@ -267,6 +271,9 @@ ValidatorSite::onSiteFetch(
             JLOG (j_.warn()) <<
                 "Unable to parse JSON response from  " <<
                 sites_[siteIdx].uri;
+
+            sites_[siteIdx].lastRefreshStatus.emplace(
+                Site::Status{clock_type::now(), ListDisposition::invalid});
         }
     }
 
```

Comment:
We currently don't record a `lastRefreshStatus` if there was an error fetching the list.
@mtrippled added an `else` for that case here (should this pr be rebased on that branch as well? :grimacing:)
https://github.com/ripple/rippled/pull/2243/files#diff-746977a6800a792fee7608383bca05deR282
I don't know if we should use `ListDisposition::invalid` or a new `ListDisposition` value?

Reply:
Yes, will add. I think `invalid` works for now.  I don't think a new name would any useful actionable information; clients would likely need to querying the publisher site either way to figure out what is going on.